source/renderer/app/components/wallet/settings/WalletSettings.js

```diff
@@ -172,8 +188,23 @@ export default class WalletSettings extends Component<Props, State> {
         styles.deleteWalletBox,
         styles.legacyWallet,
       ]);
+      const resyncWalletBoxStyles = classNames([
+        styles.resyncWalletBox,
+        styles.legacyWallet,
+      ]);
       return (
         <div className={styles.component}>
+          <BorderedBox className={resyncWalletBoxStyles}>
```

Comment:
What about creating dedicated components for Sync and Delete?

Reply:
@daniloprates please add those if you have time today.

Reply:
@daniloprates please keep in mind that we have a bit different layout for Balance vs Rewards wallets!

Reply:
@daniloprates done: https://github.com/input-output-hk/daedalus/pull/1822/commits/8b9cfab1ce1f253e31c5252d6aab4413dc2481f6